### PR TITLE
Adds shorthand alt-click for removing tanks from TTVs and adds context for it

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request

Adds shorthand alt-click for removing tanks from TTVs and adds context for it.

## Why It's Good For The Game

Hate removing these manually when refining anoms

## Changelog

:cl:
qol: Adds shorthand alt-click for removing tanks from TTVs and adds context for it
/:cl:

### CODE REVIEW

**Comment by MrMelbert** [comment]
ctrl click is a very bad hotkey on a bulky item you have to drag a lot, is alt click taken?

**Comment by carlarctg** [comment]
Not sure what youre talking about I madwe it alt click

**Comment by MrMelbert** [comment]
oh true

File: code/game/objects/items/devices/transfer_valve.dm [hunk 1]
```diff
@@ -25,11 +25,38 @@
 /obj/item/transfer_valve/Initialize(mapload)
 	. = ..()
 	RegisterSignal(src, COMSIG_ITEM_FRIED, PROC_REF(on_fried))
+	register_context()
 
 /obj/item/transfer_valve/Destroy()
 	attached_device = null
+	tank_one = null
+	tank_two = null
```
Comment (Ghommie) [inline code/game/objects/items/devices/transfer_valve.dm:33]: **Exited()** already handles this.

```suggestion
```

**Pull request merged**